theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -770,6 +770,10 @@ def tcase(self, i, f, s, b, flip, provide_shape, fd=(1, 1)):
                            mode=mode)
 
     def tcase_gi(self, i, f, o, s, b, flip, provide_shape, fd=(1, 1), expect_error=False):
+
+        if not theano.config.cxx:
+            raise SkipTest("Need cxx to test conv2d")
```

Comment:
I think this fails because it compares against CorrMM and that doesn't have python code.